src/components/example-AIAssistant.tsx

```
@@ -15,159 +15,159 @@ import GuitarRecommendation from "./example-GuitarRecommendation";
-		},
-		onToolCall: (call) => {
-			if (call.toolCall.toolName === "recommendGuitar") {
-				return "Handled by the UI";
-			}
-		},
-	});
+  const isOpen = useStore(showAIAssistant);
+  const { messages, input, handleInputChange, handleSubmit } = useChat({
+    initialMessages: [],
+    fetch: (_url, options) => {
+      const { messages } = JSON.parse(options!.body! as string);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)